### ISSUE TITLE
Sack of Holding bugging when used on Unpackager

### ISSUE THREAD

**Issue by minecraftsteve** [EnigmaticaModpacks/Enigmatica2ExpertSkyblock#622]
### Modpack Version

1.37

### Describe your issue.

TL;DR: Sack of Holding doesn't let Unpackager be placed down after being picked up, instead drops a copy of the Package Recipe Holder inside it.

When I wanted to relocate my auto Extended Crafting system, I wanted to save the effort of breaking and replacing blocks using Sack of Holding. The sack did pick up the block, but when I tried placing it back down, it generated an error message, did not place the block, and spawned a copy of the Package Recipe Holder I put inside.

![image](https://github.com/EnigmaticaModpacks/Enigmatica2ExpertSkyblock/assets/37441062/65f2f0f9-0514-4ead-b7ed-7173b5ba8eb8)

I did check latest.log and found these lines, which could explain why this was happening:

_[00:38:45] [Server thread/ERROR]: Error attempting to place block in world
java.lang.IllegalStateException: Loading data after part of a grid, this is invalid._

Thank you for your time!

### Latest Log

https://gist.github.com/minecraftsteve/ffae565df35259f1ba0fbc20c4e1d32f

### Have you modified the modpack?

No

### User Modifications

Note: I did not modify the pack before, but did update Cyclic to the newest 1.12.2 version after. The same issue persisted.

### Did the issue happen in singleplayer or on a server?

Singleplayer

### Discord Username

#steve9035

**Comment by NielsPilgaard** [comment]
Thanks for the report. You could try updating to the latest version and see if the issue persists, but to be honest I wouldn't expect the Sack of Holding to work on the Unpackager, ever :P
